csrc/scheduler/mma_utils.h

```diff
@@ -282,16 +282,19 @@ struct MatmulPattern {
 //! Traverse the fusion to find supported matmul patterns
 std::vector<MatmulPattern> findMatmulPatterns(Fusion* fusion);
 
-using MatmulProblemLayoutOpt = DataWrapperOpt<MmaLayout>;
+//! This is a vector of roles describing the inner dimension of each operand
+using MatmulOperandInnerDims = std::vector<MatmulDomain>;
```

Comment:
Should this be a `std::array<MatmulDomain, 2>`?

Reply:
I am hoping to generalize to allowing any number of operands.